src/Sarif.Multitool.Library/KustoCommand.cs

```diff
@@ -0,0 +1,180 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+
+using Kusto.Data;
+using Kusto.Data.Common;
+using Kusto.Data.Net.Client;
+
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    public class KustoCommand : CommandBase
+    {
+        private ICslQueryProvider _kustoClient;
+        private KustoOptions _options;
+
+        public int Run(KustoOptions options)
+        {
+            try
+            {
+                if (!options.Validate())
+                {
+                    return FAILURE;
+                }
+
+                _options = options;
+
+                InitializeKustoClient();
+
+                (List<Result>, List<ReportingDescriptor>) sarifResults = RetrieveResultsFromKusto();
+                var sarifLog = new SarifLog
+                {
+                    Runs = new[]
+                    {
+                    new Run
+                    {
+                        Tool = new Tool
+                        {
+                            Driver = new ToolComponent
+                            {
+                                Name = "SARIF Kusto",
```

Comment:
>ARIF Kusto [](start = 41, length = 10)

This should be 'spam' only. You can note in a property somewhere that this is spam SARIF rehydrated by the kusto command.